docs/sphinx/source/api.rst

```diff
@@ -138,7 +138,7 @@ Decomposing and combining irradiance
    irradiance.aoi_projection
    irradiance.poa_horizontal_ratio
    irradiance.beam_component
-   irradiance.globalinplane
+   irradiance.poa_components
    irradiance.grounddiffuse
```

Comment:
Is this a good opportunity to make naming more consistent, such as `irradiance.extra_radiation` and `irradiance.ground_diffuse`?

Reply:
Sure. Probably a good idea to make all API changes to a module at once.

Reply:
How about the pattern of get_extraterrestrial and get_ground_diffuse as the wrapper, and ground_diffuse_isotropic as our one and only wrapped function?  That's what I'm suggesting for sky_diffuse models. I can see the same pattern working for get_clearsky (kwarg to specify the clearsky model), get_airmass, get_iam, etc.

Many function names were simply carried over from Matlab where we wrote a library, not an object-organized package.